k8s/dependencies/kibana/elasticsearch.yaml

```diff
@@ -97,6 +97,7 @@ metadata:
   labels:
     app.kubernetes.io/name: elasticsearch
 spec:
+  type: LoadBalancer
```

Comment:
@irony / @alexanderczigler: I added this and line 112..135 to get connection to elasticsearch in dev mode.. how should we solve this long term in a better approach so that we don't have to expose ports publicly?

Reply:
Me and @Graborg reasoned that we should just keep this for now and [solve this as a separate issue]([url](https://trello.com/c/jIJc74gn/93-s%C3%A4kra-upp-kibana-elasticsearch))

Reply:
Just so I understand, is the issue that we want to be able to connect to elasticsearch running in the cluster, from our own machines?

Reply:
For me it was running the dependencies in my local kubernetes setup on my local machine and running simulator with `npm run dev` and allowing it to fill my elasticsearch with data even though it's outside the local kubernetes environment.